go/vt/vtgate/planbuilder/testdata/info_schema80_cases.json

```diff
@@ -576,17 +600,37 @@
       "QueryType": "SELECT",
       "Original": "SELECT cc.constraint_name AS 'name', cc.check_clause AS 'expression' FROM information_schema.check_constraints cc JOIN information_schema.table_constraints tc USING (constraint_schema, constraint_name) WHERE tc.table_schema = 'table_schema' AND tc.table_name = 'table_name' AND cc.constraint_schema = 'constraint_schema'",
       "Instructions": {
-        "OperatorType": "Route",
-        "Variant": "DBA",
-        "Keyspace": {
-          "Name": "main",
-          "Sharded": false
-        },
-        "FieldQuery": "select cc.constraint_name as `name`, cc.check_clause as expression from information_schema.check_constraints as cc, information_schema.table_constraints as tc where 1 != 1",
-        "Query": "select cc.constraint_name as `name`, cc.check_clause as expression from information_schema.check_constraints as cc, information_schema.table_constraints as tc where cc.constraint_schema = :__vtschemaname and tc.table_schema = :__vtschemaname and tc.table_name = :tc_table_name",
-        "SysTableTableName": "[tc_table_name:VARCHAR(\"table_name\")]",
-        "SysTableTableSchema": "[VARCHAR(\"constraint_schema\"), VARCHAR(\"table_schema\")]",
-        "Table": "information_schema.check_constraints, information_schema.table_constraints"
+        "OperatorType": "Join",
```

Comment:
Here we know that the two schemas being searched for are different - `WHERE tc.table_schema = 'table_schema' AND ... cc.constraint_schema = 'constraint_schema'`. Merging these two was wrong.